conf/config.go

```diff
@@ -27,6 +27,9 @@ var (
 	ignoredGIs  = getIgnoredGIs()
 
 	TrustedProxies = parseTrustedProxies()
+
+	// Eclypsium registration token, passed into osie
+	EclypsiumToken = env.Get("ECLYPSIUM_TOKEN", "")
```

Comment:
just `env.Get("ECLYPSIUM_TOKEN")` is best.

Reply:
ack.